includes/class-amp-theme-support.php

```diff
@@ -1514,6 +1513,11 @@ public static function ensure_required_markup( Document $dom, $script_handles =
 			$superfluous_script_handles = array_diff( $superfluous_script_handles, [ 'amp-carousel' ] );
 		}
 
+		// When opting-in to POST forms, omit the amp-form component entirely since it blocks submission.
+		if ( amp_is_allowing_native_post_forms() && $dom->xpath->query( '//form[ @method = "post" and @action ]' )->length > 0 ) {
```

Comment:
Would the value of `@method` be case sensitive here?

Reply:
Good catch. Fixed in dfb5223b5.